tests/stress_tests/scripts/get_testdata.py

```diff
@@ -94,6 +107,9 @@ def main():
     parser.add_argument('--test_conf', required=True, type=Path,
                         help='Path to a test config .xml file containing models '
                              'which will be downloaded and converted to IRs via OMZ.')
+    parser.add_argument('--test_framework', required=False, default=None,
```

Comment:
Why framework selection is needed? Can we autodetect it?

Reply:
Yes, we can, I done it

Reply:
I'd expect you to remove that option then.